parser-typechecker/src/Unison/Runtime/Builtin.hs

```diff
@@ -242,10 +242,6 @@ gei = cmpopb LEQI Ty.intRef
 gtn = cmpopn LEQN Ty.intRef
 gen = cmpopb LEQN Ty.intRef
 
-neqi, neqn :: Var v => SuperNormal v
-neqi = cmpopn EQLI Ty.intRef
-neqn = cmpopn EQLN Ty.intRef
```

Comment:
Deleted these as there isn't currently a `Nat.!=` or `Int.!=` builtin and they weren't being used.